working-groups/wg-dm-portal.md

```diff
@@ -0,0 +1,21 @@
+# DM Portal Working Group
+
+## Goals
+
+Build a small web app to help generate the DM config file easier.
+
+## Meetings
+
+To be determined.
```

Comment:
please add the public meeting note and zoom urls.

Reply:
ok, I will add it later after it is determined.

Reply:
@zz-jason already added it, PTAL